### ISSUE TITLE
PR #6889 breaks CentOS 7.6 build

### ISSUE THREAD

**Issue by ignacionr** [symless/synergy-core#6904]
### Operating Systems ###

CentOS 7.6

### Synergy Version ###

trunk

### Steps to Reproduce ###

Copy the CI Azure steps on a Dockerfile, such as follows.

```dockerfile
FROM symless/synergy-core:centos7.6

WORKDIR /app
RUN git clone  --recurse-submodules https://github.com/symless/synergy-core

WORKDIR /app/synergy-core
RUN       mkdir build-release
WORKDIR /app/synergy-core/build-release
RUN       cmake3 -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX:PATH=$(pwd)/rpm/BUILDROOT/usr ..;
RUN       . ./version
RUN       make
```

Then try to build the container.

**Comment by andreacndn** [comment]
SYNERGY-734

**Comment by ignacionr** [comment]
This is explained [here](https://github.com/symless/synergy-core/pull/6889#issuecomment-757512490) on #6889

**Comment by andreacndn** [comment]
SYNERGY-766

**Comment by SerhiiGadzhilov** [comment]
Fixed under #6937
